### PR TITLE
[MRG] load surf data in view_surf

### PR DESCRIPTION
**Pull request by jeromedockes**
otherwise if it is provided as a file path the function fails e.g.
```
from nilearn import datasets, plotting
fsaverage = datasets.fetch_surf_fsaverage()
plotting.view_surf(fsaverage['pial_left'], fsaverage['sulc_left'],
                   cmap='Greys', threshold=None, symmetric_cmap=False)

```

### CODE REVIEW

**Comment by kchawla-pi** [comment]
Hey, this is cool. Would you document the reason for this PR in the first comment, so there is a record of the problem or even better file an issue and link it here?

**Comment by kchawla-pi** [comment]
I was about to merge it, we should add Fix in whats_new .

File: nilearn/plotting/html_surface.py [hunk 1]
```diff
@@ -297,9 +297,10 @@ def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
     if surf_map is None:
         surf_map = np.ones(len(surf_mesh[0]))
     if surf_map is not None:
```
Comment (kchawla-pi) [inline nilearn/plotting/html_surface.py]: SHouldn't we just make this `else:`?

Comment (jeromedockes) [reply nilearn/plotting/html_surface.py]: :laughing: why make things simple when you can make them complicated?

**Comment by kchawla-pi** [comment]
WHy did none of the tests catch this earlier? Do we have a test that catches this now?

**Comment by jeromedockes** [comment]
> Do we have a test that catches this now?

yes it is added in test_html_surface.py in this pr

**Comment by jeromedockes** [comment]
@kchawla-pi  do you have more comments or can we merge?

**Pull request merged**